module/updatable_configs/validation/require_approvals_for_seal_construction.go

```diff
@@ -0,0 +1,35 @@
+package validation
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/module/defaults"
+)
+
+// ValidateRequireApprovals validates the given value against the default of other values.
+func ValidateRequireApprovals(requiredApprovalsForSealConstruction uint) error {
+	// NOTE: validating against the default value. Even though we take in new values from flag, but
+	// in practice we never use this flag, because it would need a coordination of all nodes updating
+	// the flag, which will most likely done by updating a new image with a new default value.
+	return validateRequireApprovals(
+		requiredApprovalsForSealConstruction,
+		defaults.DefaultRequiredApprovalsForSealValidation,
+		// chunks.DefaultChunkAssignmentAlpha,
+		3,
```

Comment:
## ⚠️ 
I think **we can not use default values**. This is because the chunk alpha can be changed as a command line parameter: 
https://github.com/onflow/flow-go/blob/3e92041711fa82d322ef803fb3c88fd42f599b18/cmd/consensus/main.go#L148-L150

Validation must consider the current values (and not the default values, which could be different). 

For example, assume that we set `chunkAlpha` to 10. Then, the admin too will refuse to set required `requiredApprovalsForSealConstruction` to 5, despite this being a perfectly fine value.

Reply:
This is not easy to do, because we want the `DefaultRequiredApprovalsForSealConstruction` to be updatable by any module from any file.

Why?

Because the admin tool doesn't have access to the NodeBuilder instance to access this variable, therefore, we had to make it as a global singleton. 

But we don't have to, if we can create a stateful config module, and pass it to the admin tool. 

That way we can read the latest values, and create the stateful config module with it, and pass to the admin tool. 

This would be the most clean way.